dex/src/test/scala/com/wavesplatform/dex/settings/BaseSettingsSpecification.scala

```diff
@@ -18,6 +18,7 @@ class BaseSettingsSpecification extends FlatSpec {
        |  mode = percent
        |  dynamic {
        |    base-fee = 300000
+       |    zero-maker-double-taker = false
```

Comment:
For tests it's better to specify a value different from application.conf's

Reply:
Done